packages/chains/src/bsc/redemptionStrategies.ts

```diff
@@ -100,7 +100,17 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
     RedemptionStrategyContract.UniswapLpTokenLiquidator,
     underlying(assets, assetSymbols.HAY),
   ],
+  [underlying(assets, assetSymbols["sAMM-HAY/BUSD"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
+    underlying(assets, assetSymbols.BUSD),
+  ],
+  [underlying(assets, assetSymbols["vAMM-HAY/ankrBNB"])]: [
+    RedemptionStrategyContract.UniswapLpTokenLiquidator,
```

Comment:
are we sure these pairs can be liquidated with the `UniswapLpTokenLiquidator` ?

Reply:
Yeah, now that I realise it, no they cannot...

Reply:
https://github.com/Midas-Protocol/contracts/blob/development/contracts/test/liquidators/UniswapLikeLpTokenLiquidatorTest.t.sol

Reply:
They can actually